Add the Updated ScreenShot of the Application's Logout Screen

aakarshvats638 opened the pull request:
### Description
Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.
Fix=#Add the Updated ScreenShot of the Application's Logout Screen
Fixes # 1046

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)



### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.


### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [x] I have written Kotlin Docs whenever is applicable


**Code/Quality Assurance Only**
- [x] My changes generate no new warnings
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

aakarshvats638 committed:
Logout screen SS updated

aakarshvats638 committed:
abcd

aakarshvats638 committed:
Checking

aakarshvats638 commented:
#1044 solved

vj-codes reviewed (changes requested):
@aakarshvats638 welcome to the community and congrats on your first PR!
Can you please update these changes -
1) Follow our [Commit MEssage Style Guide](https://github.com/anitab-org/mentorship-android/wiki/Commit-Message-Style-Guide) for the PR title and commit message.
2) The number of files changed for this PR is showing 253 whereas only one file(readme) should be changed to fix the issue.
3) Mention the issue number in the `Fixes` part of PR description.
Just these small changes and you're done!Do update if you have any doubts:)

aakarshvats638 commented:
Actually I have changed only one file i.e Readme and have added the screenshot In screenshot folder ,i don't know why it is showing that I have changed 253 files.

techno-disaster commented:
> Actually I have changed only one file i.e Readme and have added the screenshot In screenshot folder ,i don't know why it is showing that I have changed 253 files.

you named your branch ```aakarsh/metorship-android ``` so everyting is going in [this](https://github.com/aakarshvats638/mentorship-android/tree/aakarsh/metorship-android/mentorship-android-aakarsh-metorship-android) folder

vj-codes commented:
@aakarshvats638 hey any updates?

aakarshvats638 commented:
Actually, my exams were going on, I'll do the required changes by tomorrow.
